### ISSUE TITLE
Various test failures with tomlkit 0.13.1 and newer

### ISSUE THREAD

**Issue by mweinelt** [remarshal-project/remarshal#50]
Hi!

We're using remarshal 0.18.0 in nixpkgs and trying to upgrade tomlkit from 0.12.5 to a newer version.

- 0.13.0 works
- 0.13.1 and 0.13.2 fail the testsuite

```pyb
_________________________ TestRemarshal.test_toml2toml _________________________

self = <tests.test_remarshal.TestRemarshal object at 0x7ffff66bd9a0>
convert_and_read = functools.partial(<function _convert_and_read at 0x7ffff65df1a0>, output_filename='/build/pytest-of-nixbld/pytest-0/test_toml2toml0/4d8f7acf1408f5454367f7dd311d3441')

    def test_toml2toml(self, convert_and_read) -> None:
        output = convert_and_read("example.toml", "toml", "toml")
        reference = read_file("example.toml")
>       assert toml_signature(output) == toml_signature(reference)
E       assert ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob=1979-05-27T07:32:00Z', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"'] == ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', '[servers]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob=1979-05-27T07:32:00Z', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"']
E         
E         At index 7 diff: 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."' != '[servers]'
E         Right contains one more item: 'title="TOMLExample"'
E         
E         Full diff:
E           [
E               '[[products]]',
E               '[[products]]',
E               '[clients]',
E               '[database]',
E               '[owner]',
E               '[servers.alpha]',
E               '[servers.beta]',
E         -     '[servers]',
E               'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."',
E               'color="gray"',
E               'connection_max=5000',
E               'country="中国"',
E               'data=[["gamma","delta"],[1,2]]',
E               'dc="eqdc10"',
E               'dc="eqdc10"',
E               'dob=1979-05-27T07:32:00Z',
E               'enabled=true',
E               'ip="10.0.0.1"',
E               'ip="10.0.0.2"',
E               'name="Hammer"',
E               'name="Nail"',
E               'name="TomPreston-Werner"',
E               'organization="GitHub"',
E               'ports=[8001,8001,8002]',
E               'server="192.168.1.1"',
E               'sku=284758393',
E               'sku=738594937',
E               'title="TOMLExample"',
E           ]

tests/test_remarshal.py:148: AssertionError
_________________________ TestRemarshal.test_json2toml _________________________

self = <tests.test_remarshal.TestRemarshal object at 0x7ffff66bfb00>
convert_and_read = functools.partial(<function _convert_and_read at 0x7ffff65df1a0>, output_filename='/build/pytest-of-nixbld/pytest-0/test_json2toml0/2dee661d7da41fabb251faacae57c818')

    def test_json2toml(self, convert_and_read) -> None:
        output = convert_and_read("example.json", "json", "toml").decode("utf-8")
        reference = read_file("example.toml").decode("utf-8")
        output_sig = toml_signature(output)
        # The date in 'example.json' is a string.
        reference_sig = toml_signature(
            reference.replace("1979-05-27T07:32:00Z", '"1979-05-27T07:32:00+00:00"')
        )
>       assert output_sig == reference_sig
E       assert ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob="1979-05-27T07:32:00+00:00"', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"'] == ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', '[servers]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob="1979-05-27T07:32:00+00:00"', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"']
E         
E         At index 7 diff: 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."' != '[servers]'
E         Right contains one more item: 'title="TOMLExample"'
E         
E         Full diff:
E           [
E               '[[products]]',
E               '[[products]]',
E               '[clients]',
E               '[database]',
E               '[owner]',
E               '[servers.alpha]',
E               '[servers.beta]',
E         -     '[servers]',
E               'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."',
E               'color="gray"',
E               'connection_max=5000',
E               'country="中国"',
E               'data=[["gamma","delta"],[1,2]]',
E               'dc="eqdc10"',
E               'dc="eqdc10"',
E               'dob="1979-05-27T07:32:00+00:00"',
E               'enabled=true',
E               'ip="10.0.0.1"',
E               'ip="10.0.0.2"',
E               'name="Hammer"',
E               'name="Nail"',
E               'name="TomPreston-Werner"',
E               'organization="GitHub"',
E               'ports=[8001,8001,8002]',
E               'server="192.168.1.1"',
E               'sku=284758393',
E               'sku=738594937',
E               'title="TOMLExample"',
E           ]

tests/test_remarshal.py:201: AssertionError
_______________________ TestRemarshal.test_msgpack2toml ________________________

self = <tests.test_remarshal.TestRemarshal object at 0x7ffff65c7230>
convert_and_read = functools.partial(<function _convert_and_read at 0x7ffff65df1a0>, output_filename='/build/pytest-of-nixbld/pytest-0/test_msgpack2toml0/b6e7f763cfe2bd0c11607dc6fd2068a6')

    def test_msgpack2toml(self, convert_and_read) -> None:
        output = convert_and_read("example.msgpack", "msgpack", "toml")
        reference = read_file("example.toml")
>       assert toml_signature(output) == toml_signature(reference)
E       assert ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob=1979-05-27T07:32:00Z', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"'] == ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', '[servers]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob=1979-05-27T07:32:00Z', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"']
E         
E         At index 7 diff: 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."' != '[servers]'
E         Right contains one more item: 'title="TOMLExample"'
E         
E         Full diff:
E           [
E               '[[products]]',
E               '[[products]]',
E               '[clients]',
E               '[database]',
E               '[owner]',
E               '[servers.alpha]',
E               '[servers.beta]',
E         -     '[servers]',
E               'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."',
E               'color="gray"',
E               'connection_max=5000',
E               'country="中国"',
E               'data=[["gamma","delta"],[1,2]]',
E               'dc="eqdc10"',
E               'dc="eqdc10"',
E               'dob=1979-05-27T07:32:00Z',
E               'enabled=true',
E               'ip="10.0.0.1"',
E               'ip="10.0.0.2"',
E               'name="Hammer"',
E               'name="Nail"',
E               'name="TomPreston-Werner"',
E               'organization="GitHub"',
E               'ports=[8001,8001,8002]',
E               'server="192.168.1.1"',
E               'sku=284758393',
E               'sku=738594937',
E               'title="TOMLExample"',
E           ]

tests/test_remarshal.py:220: AssertionError
_________________________ TestRemarshal.test_yaml2toml _________________________

self = <tests.test_remarshal.TestRemarshal object at 0x7ffff6604620>
convert_and_read = functools.partial(<function _convert_and_read at 0x7ffff65df1a0>, output_filename='/build/pytest-of-nixbld/pytest-0/test_yaml2toml0/4bf4fe5dfa750c26bcdbed1cbec867c7')

    def test_yaml2toml(self, convert_and_read) -> None:
        output = convert_and_read("example.yaml", "yaml", "toml")
        reference = read_file("example.toml")
>       assert toml_signature(output) == toml_signature(reference)
E       assert ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob=1979-05-27T07:32:00Z', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"'] == ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', '[servers]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob=1979-05-27T07:32:00Z', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"']
E         
E         At index 7 diff: 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."' != '[servers]'
E         Right contains one more item: 'title="TOMLExample"'
E         
E         Full diff:
E           [
E               '[[products]]',
E               '[[products]]',
E               '[clients]',
E               '[database]',
E               '[owner]',
E               '[servers.alpha]',
E               '[servers.beta]',
E         -     '[servers]',
E               'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."',
E               'color="gray"',
E               'connection_max=5000',
E               'country="中国"',
E               'data=[["gamma","delta"],[1,2]]',
E               'dc="eqdc10"',
E               'dc="eqdc10"',
E               'dob=1979-05-27T07:32:00Z',
E               'enabled=true',
E               'ip="10.0.0.1"',
E               'ip="10.0.0.2"',
E               'name="Hammer"',
E               'name="Nail"',
E               'name="TomPreston-Werner"',
E               'organization="GitHub"',
E               'ports=[8001,8001,8002]',
E               'server="192.168.1.1"',
E               'sku=284758393',
E               'sku=738594937',
E               'title="TOMLExample"',
E           ]

tests/test_remarshal.py:281: AssertionError
_________________________ TestRemarshal.test_cbor2toml _________________________

self = <tests.test_remarshal.TestRemarshal object at 0x7ffff6604ce0>
convert_and_read = functools.partial(<function _convert_and_read at 0x7ffff65df1a0>, output_filename='/build/pytest-of-nixbld/pytest-0/test_cbor2toml0/56c34b95d89b9cd50374f87c69cad1eb')

    def test_cbor2toml(self, convert_and_read) -> None:
        output = convert_and_read("example.cbor", "cbor", "toml")
        reference = read_file("example.toml")
        output_sig = toml_signature(output)
        reference_sig = toml_signature(reference)
>       assert output_sig == reference_sig
E       assert ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob=1979-05-27T07:32:00Z', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"'] == ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', '[servers]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob=1979-05-27T07:32:00Z', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"']
E         
E         At index 7 diff: 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."' != '[servers]'
E         Right contains one more item: 'title="TOMLExample"'
E         
E         Full diff:
E           [
E               '[[products]]',
E               '[[products]]',
E               '[clients]',
E               '[database]',
E               '[owner]',
E               '[servers.alpha]',
E               '[servers.beta]',
E         -     '[servers]',
E               'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."',
E               'color="gray"',
E               'connection_max=5000',
E               'country="中国"',
E               'data=[["gamma","delta"],[1,2]]',
E               'dc="eqdc10"',
E               'dc="eqdc10"',
E               'dob=1979-05-27T07:32:00Z',
E               'enabled=true',
E               'ip="10.0.0.1"',
E               'ip="10.0.0.2"',
E               'name="Hammer"',
E               'name="Nail"',
E               'name="TomPreston-Werner"',
E               'organization="GitHub"',
E               'ports=[8001,8001,8002]',
E               'server="192.168.1.1"',
E               'sku=284758393',
E               'sku=738594937',
E               'title="TOMLExample"',
E           ]

tests/test_remarshal.py:302: AssertionError
=========================== short test summary info ============================
FAILED tests/test_remarshal.py::TestRemarshal::test_toml2toml - assert ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob=1979-05-27T07:32:00Z', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"'] == ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', '[servers]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob=1979-05-27T07:32:00Z', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"']
  
  At index 7 diff: 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."' != '[servers]'
  Right contains one more item: 'title="TOMLExample"'
  
  Full diff:
    [
        '[[products]]',
        '[[products]]',
        '[clients]',
        '[database]',
        '[owner]',
        '[servers.alpha]',
        '[servers.beta]',
  -     '[servers]',
        'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."',
        'color="gray"',
        'connection_max=5000',
        'country="中国"',
        'data=[["gamma","delta"],[1,2]]',
        'dc="eqdc10"',
        'dc="eqdc10"',
        'dob=1979-05-27T07:32:00Z',
        'enabled=true',
        'ip="10.0.0.1"',
        'ip="10.0.0.2"',
        'name="Hammer"',
        'name="Nail"',
        'name="TomPreston-Werner"',
        'organization="GitHub"',
        'ports=[8001,8001,8002]',
        'server="192.168.1.1"',
        'sku=284758393',
        'sku=738594937',
        'title="TOMLExample"',
    ]
FAILED tests/test_remarshal.py::TestRemarshal::test_json2toml - assert ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob="1979-05-27T07:32:00+00:00"', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"'] == ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', '[servers]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob="1979-05-27T07:32:00+00:00"', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"']
  
  At index 7 diff: 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."' != '[servers]'
  Right contains one more item: 'title="TOMLExample"'
  
  Full diff:
    [
        '[[products]]',
        '[[products]]',
        '[clients]',
        '[database]',
        '[owner]',
        '[servers.alpha]',
        '[servers.beta]',
  -     '[servers]',
        'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."',
        'color="gray"',
        'connection_max=5000',
        'country="中国"',
        'data=[["gamma","delta"],[1,2]]',
        'dc="eqdc10"',
        'dc="eqdc10"',
        'dob="1979-05-27T07:32:00+00:00"',
        'enabled=true',
        'ip="10.0.0.1"',
        'ip="10.0.0.2"',
        'name="Hammer"',
        'name="Nail"',
        'name="TomPreston-Werner"',
        'organization="GitHub"',
        'ports=[8001,8001,8002]',
        'server="192.168.1.1"',
        'sku=284758393',
        'sku=738594937',
        'title="TOMLExample"',
    ]
FAILED tests/test_remarshal.py::TestRemarshal::test_msgpack2toml - assert ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob=1979-05-27T07:32:00Z', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"'] == ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', '[servers]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob=1979-05-27T07:32:00Z', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"']
  
  At index 7 diff: 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."' != '[servers]'
  Right contains one more item: 'title="TOMLExample"'
  
  Full diff:
    [
        '[[products]]',
        '[[products]]',
        '[clients]',
        '[database]',
        '[owner]',
        '[servers.alpha]',
        '[servers.beta]',
  -     '[servers]',
        'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."',
        'color="gray"',
        'connection_max=5000',
        'country="中国"',
        'data=[["gamma","delta"],[1,2]]',
        'dc="eqdc10"',
        'dc="eqdc10"',
        'dob=1979-05-27T07:32:00Z',
        'enabled=true',
        'ip="10.0.0.1"',
        'ip="10.0.0.2"',
        'name="Hammer"',
        'name="Nail"',
        'name="TomPreston-Werner"',
        'organization="GitHub"',
        'ports=[8001,8001,8002]',
        'server="192.168.1.1"',
        'sku=284758393',
        'sku=738594937',
        'title="TOMLExample"',
    ]
FAILED tests/test_remarshal.py::TestRemarshal::test_yaml2toml - assert ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob=1979-05-27T07:32:00Z', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"'] == ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', '[servers]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob=1979-05-27T07:32:00Z', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"']
  
  At index 7 diff: 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."' != '[servers]'
  Right contains one more item: 'title="TOMLExample"'
  
  Full diff:
    [
        '[[products]]',
        '[[products]]',
        '[clients]',
        '[database]',
        '[owner]',
        '[servers.alpha]',
        '[servers.beta]',
  -     '[servers]',
        'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."',
        'color="gray"',
        'connection_max=5000',
        'country="中国"',
        'data=[["gamma","delta"],[1,2]]',
        'dc="eqdc10"',
        'dc="eqdc10"',
        'dob=1979-05-27T07:32:00Z',
        'enabled=true',
        'ip="10.0.0.1"',
        'ip="10.0.0.2"',
        'name="Hammer"',
        'name="Nail"',
        'name="TomPreston-Werner"',
        'organization="GitHub"',
        'ports=[8001,8001,8002]',
        'server="192.168.1.1"',
        'sku=284758393',
        'sku=738594937',
        'title="TOMLExample"',
    ]
FAILED tests/test_remarshal.py::TestRemarshal::test_cbor2toml - assert ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob=1979-05-27T07:32:00Z', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"'] == ['[[products]]', '[[products]]', '[clients]', '[database]', '[owner]', '[servers.alpha]', '[servers.beta]', '[servers]', 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."', 'color="gray"', 'connection_max=5000', 'country="中国"', 'data=[["gamma","delta"],[1,2]]', 'dc="eqdc10"', 'dc="eqdc10"', 'dob=1979-05-27T07:32:00Z', 'enabled=true', 'ip="10.0.0.1"', 'ip="10.0.0.2"', 'name="Hammer"', 'name="Nail"', 'name="TomPreston-Werner"', 'organization="GitHub"', 'ports=[8001,8001,8002]', 'server="192.168.1.1"', 'sku=284758393', 'sku=738594937', 'title="TOMLExample"']
  
  At index 7 diff: 'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."' != '[servers]'
  Right contains one more item: 'title="TOMLExample"'
  
  Full diff:
    [
        '[[products]]',
        '[[products]]',
        '[clients]',
        '[database]',
        '[owner]',
        '[servers.alpha]',
        '[servers.beta]',
  -     '[servers]',
        'bio="GitHubCofounder&CEO\\nLikestatertotsandbeer."',
        'color="gray"',
        'connection_max=5000',
        'country="中国"',
        'data=[["gamma","delta"],[1,2]]',
        'dc="eqdc10"',
        'dc="eqdc10"',
        'dob=1979-05-27T07:32:00Z',
        'enabled=true',
        'ip="10.0.0.1"',
        'ip="10.0.0.2"',
        'name="Hammer"',
        'name="Nail"',
        'name="TomPreston-Werner"',
        'organization="GitHub"',
        'ports=[8001,8001,8002]',
        'server="192.168.1.1"',
        'sku=284758393',
        'sku=738594937',
        'title="TOMLExample"',
    ]
========================= 5 failed, 86 passed in 1.17s =========================
```

**Comment by dbohdan** [comment]
Confirmed. I'll look into this and bump the minimum version of TOML Kit to 0.13.2.

**Comment by dbohdan** [comment]
This is the cause upstream: https://github.com/python-poetry/tomlkit/pull/378. I am glad it has been fixed. The empty tables were TOML Kit's signature style :-), but there was never a need for them, and they added noise.

**Comment by dbohdan** [comment]
I have published version 0.19.0 as a Git tag and a release on GitHub and on PyPI. Please try it and see if it fixes your problem.

**Comment by mweinelt** [comment]
Thank you, things are looking good on my end with 0.19.0 and 0.13.2!
